client/src/components/LoginForm/LoginForm.jsx

```diff
@@ -2,7 +2,6 @@
 import React, {Component} 						from 'react';
```

Comment:
I think it's really worth while to split this huge file to smaller components.. not critical though